### ISSUE TITLE
Arguman.org is down since a few days 

### ISSUE THREAD

**Issue by swergas** [arguman/arguman.org#399]
Hi,
The arguman.org website and its local versions seem to be down since a few days

**Comment by sercanarga** [comment]
I think argument.org is dead. :(

**Comment by sercanarga** [comment]
I think arguman.org is dead. :(

**Comment by fuu0** [comment]
If somebody has connections to the person behind arguman.org:
I have infrastructure and the will to host OSS like that.
Please contact me under fuu0+arumanorg@posteo.net - don't let it die, arguing and the tools to argue is more impotent than ever before!

**Comment by ghost** [comment]
looks to have been hacked to promote some pernicious chrome extension.

**Comment by Venryx** [comment]
Yes, the domain seems to have been bought by someone who turned it into an ad page or something.

The source-code on the domain contains a reference to `0redirc.com`; googling it gives some results, including: https://www.reddit.com/r/webdev/comments/nfq0bm/fighting_back_against_scam_texts

It's interesting that the source-code also contains reference to `swfobject.js`, which is probably used to get more information about the user's computer to "fingerprint" them, ie. identify the user for long-term tracking. This is supported by this section in the JS code:
```
debugInfo = gl.getExtension('WEBGL_debug_renderer_info');
vendor = gl.getParameter(debugInfo.UNMASKED_VENDOR_WEBGL);
renderer = gl.getParameter(debugInfo.UNMASKED_RENDERER_WEBGL);
gstr = vendor+' - '+renderer;
 
//limit & encode
gpu = '&gpu='+encodeURI(gstr.substring(0,200));

[...]

window.location.replace(a + b + c+vs+ds+sl+os+nos+swfV+inside_frame+sc+gpu);
```

**Comment by plehman2000** [comment]
Unfortunate that noone really uses it. Difficult and in depth debating is not a popular pasttime. Hopefully we can change that in the future

**Comment by raisercostin** [comment]
I assume the content was also lost. Right? This screams for #ownyourdata and github as database archive for arguments.

**Comment by joaosimuai** [comment]
Wow. I just discovered it today, ahaha. Something must be done, as it is actively misleading users

**Comment by jayaddison** [comment]
The arguman service is available again at https://arguman.org - although only in Türkçe (Turkish) language so far I think.  It would be impressive if historic debate data could be restored (I have no connection with the site operators so I don't know whether backups are available).

**Comment by marienhof** [comment]
Let's resurrect the English version! Even if in a tabula rasa state.

**Comment by mirkan1** [comment]
how come nobody has a stable version?

**Comment by sercanarga** [comment]
current domain owner should contact me.

**Comment by sercanarga** [comment]
arguman.org has been maintained all this time by a turkish developer. in 2021, this is one of the last posts from him;

<img width="604" alt="Screenshot 2024-02-13 at 9 01 11 PM" src="https://github.com/arguman/arguman.org/assets/24681882/4c00cea4-0de8-4c6b-a477-087f8e1ea62e">

**Comment by mirkan1** [comment]
I recalled that he made us work on a website about a book of Nisanyan, Today I understood how you felt.
![20240509_225602.jpg](https://github.com/arguman/arguman.org/assets/32144322/60b90575-ccd5-4564-b1c9-c992435aeef5)
Nonetheless, this repository deserves to be alive again
